tests/Sdk/Unit/Extension/ValueResolver/NamespaceValueResolverTest.php

```diff
@@ -13,6 +13,9 @@
 use SprykerSdk\SdkContracts\Entity\ContextInterface;
 use SprykerSdk\SdkContracts\ValueReceiver\ValueReceiverInterface;
 
+/**
+ * @group Test1
+ */
```

Comment:
Wrong list of groups, please provide correct one.

Reply:
fixed